packages/features/bookings/Booker/components/DatePicker.tsx

```diff
@@ -64,6 +64,11 @@ export const DatePicker = ({
       selected={dayjs(selectedDate)}
       weekStart={weekdayToWeekIndex(event?.data?.users?.[0]?.weekStart)}
       slots={schedule?.data?.slots}
+      minDate={
+        schedule?.data?.slots && Object.keys(schedule?.data?.slots).length
```

Comment:
Contrary to this PR - https://github.com/calcom/cal.com/pull/15840 -

I believe using the earliest slot instead of yesterday as the minDate would be better. Because if suppose there were more slots available even before yesterday, then? (I agree that ideally this will not happen, but just as a sanity check).

Reply:
@Souptik2001 , this allows user to select slot from 1st Day of the month irrespective of current date. This should not happen, as we should not allow booking in past.

Reply:
Cross-posting this comment here - https://github.com/calcom/cal.com/issues/15833#issuecomment-2241249918